src/components/calculator/SavedDataSelector.tsx

```diff
@@ -38,6 +38,7 @@ export const SavedDataSelector: React.FunctionComponent<{
       props.setter({
         ...props.currentData,
         ...foundData,
+        scenarioName: value,
```

Comment:
ResetForm doesn't currently clear the scenario dropdown, which seems wrong; could you fix that? I think the most straightforward way is use a GenericSelectControl below.

Reply:
Yep. Will do. I've got an open issue about that. Now seems like a good time to fix it.

Reply:
GenericSelectControl is meant to be used for selects that have a multiplier attached. 

I'm also not clear how using a generic select control would cause the form to reset when you press the reset button. How do those two connect?

Reply:
Mmm, yeah, GenericSelectControl wants the options to be FormValue.

I guess I don't much care whether you use GenericSelect or update this select (I thought GenericSelect would reduce the amount of boilerplate around styling, but maybe that's not true). The relevant part is setting `value` from `calculatorData` so that updating `calculatorData.scenario` updates the dropdown.

Reply:
Oh I see. You're saying it should be a two-way connection. Is that right?

Hmmm. I was treating it as a one-way connection where the calculator just holds the current label of the scenario. It's not attempting to hold the actual "scenario" (which would include storing the scenario key). If we want to do that, we could. Is that what you'd suggest?

What I was thinking is just adding some code to the reset button that clears the scenario dropdown. Would that work too?

Reply:
That doesn't make sense if scenarioName is part of calculatorData. Either it's part of calculatorData, in which case it should be controlled by calculatorData, or it's not, in which case it should NEVER be part of it.

I'm fine with either. I slightly prefer not putting it into calculatorData (since that will cause share-by-link to show "you selected X scenario", which the recipient did not.

Reply:
Ah got it. I'll remove it from calculator data then.